docs/guides/dynamic-loading-catalogs.rst

```diff
@@ -39,12 +39,14 @@ to ECMAScript. We need to install ``babel-plugin-syntax-dynamic-import`` and
    // .babelrc
    {
      "plugins": [
-       "syntax-dynamic-import"
+       "syntax-dynamic-import",
+       "transform-class-properties"
      ],
      "env": {
        "test": {
          "plugins": [
-           "dynamic-import-node"
+           "dynamic-import-node",
+           "transform-class-properties"
```

Comment:
The plugin isn't required here. It's already on line 43, which adds it in all environments.

Reply:
Oh of course, my bad. I will fix!